common3.json

```diff
@@ -6971,375 +6971,375 @@
-      "text" : "Speed drop: 対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」を持っている"
+      "text" : "Speed drop: Target's Pokémon's ability is Clear Body, Shield Dust or White Smoke"
     },
     {
       "id" : 61811,
-      "text" : "Guaranteed Speed Drop: 相手側に自分側より素早いポケモンがいる"
+      "text" : "Guaranteed Speed Drop: Opponent side has Pokémon faster than user's side"
     },
     {
       "id" : 61812,
-      "text" : "Guaranteed Speed Drop: 対象の素早さがこれ以上下がらない"
+      "text" : "Guaranteed Speed Drop: Target's Speed will not drop further"
     },
     {
       "id" : 61813,
-      "text" : "Guaranteed Speed Drop: 対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」を持っている"
+      "text" : "Guaranteed Speed Drop: Target Pokémon's Ability is Clear Body, Shield Dust or White Smoke"
     },
     {
       "id" : 61814,
-      "text" : "Icy Wind　相手側に自分側より素早いポケモンがいる"
+      "text" : "Icy Wind　Opponent side has Pokémon faster than user's side"
     },
     {
       "id" : 61815,
-      "text" : "Icy Wind　ダブルバトルである"
+      "text" : "Icy Wind　Is a Double Battle"
     },
     {
       "id" : 61816,
-      "text" : "Icy Wind　対象の素早さが自分側より低い"
+      "text" : "Icy Wind　Target's speed is lower than User"
     },
     {
       "id" : 61817,
-      "text" : "Icy Wind　対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」を持っている"
+      "text" : "Icy Wind　Target Pokémon's Ability is Clear Body, Shield Dust or White Smoke"
     },
     {
       "id" : 61818,
-      "text" : "Sp.atk drop: 対象の特攻がこれ以上下がらない"
+      "text" : "Sp.atk drop: Target's Sp Atk will not drop further"
     },
     {
       "id" : 61819,
-      "text" : "Sp.atk drop: 対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」を持っている"
+      "text" : "Sp.atk drop: Target Pokémon's Ability is Clear Body, Shield Dust or White Smoke"
     },
     {
       "id" : 61820,
-      "text" : "Sp.def drop: 対象の特防がこれ以上下がらない"
+      "text" : "Sp.def drop: Target's Sp Def will not drop further"
     },
     {
       "id" : 61821,
-      "text" : "Sp.def drop: 対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」を持っている"
+      "text" : "Sp.def drop: Target Pokémon's Ability is Clear Body, Shield Dust or White Smoke"
     },
     {
       "id" : 61822,
-      "text" : "Accuracy drop: 対象の命中がこれ以上下がらない"
+      "text" : "Accuracy drop: Target's Accuracy will not drop further"
     },
     {
       "id" : 61823,
-      "text" : "Accuracy drop: 対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」「するどいめ」を持っている"
+      "text" : "Accuracy drop: Target Pokémon's Ability is Clear Body, Shield Dust, White Smoke or Keen Eye"
     },
     {
       "id" : 61824,
-      "text" : "Mud-slap　追加効果で必ず命中率を下げることが出来る"
+      "text" : "Mud-slap: Accuracy rate will always be lowered as an additional effect"
     },
     {
       "id" : 61825,
-      "text" : "Mud-slap　対象の命中がこれ以上下がらない"
+      "text" : "Mud-slap: Target's Accuracy will not drop further"
     },
     {
       "id" : 61826,
-      "text" : "Mud-slap　対象のポケモンが特性「クリアボディ」「りんぷん」「しろいけむり」「するどいめ」を持っている"
+      "text" : "Mud-slap: Target Pokémon's Ability is 「Clear Body」「Shield Dust」「White Smoke」「Keen Eye」を持っている"
```

Comment:
This one is weird, first off what are these right angle...thingies? Sounds like they should be replaced and what about this untranslated bit?

Reply:
Missed that one sorry. Those Right angles are Japanese Brackets.